### PR TITLE
Update Gardener endpoints #28

### PR DESCRIPTION
**Pull request by StormyW618**
Added put request and Try Catch Block

### CODE REVIEW

File: backend/server.ts [hunk 1]
```diff
@@ -30,4 +30,20 @@ app.get("/gardens", async (req:Request , res:Response) => {
   res.send(gardens)
 })
 
+//updates a garden
+app.put("/gardens/:id", async (req:Request, res: Response) => {
+  const GardenName = req.params.recipeName
```
Comment (NestorJMartinez) [inline backend/server.ts]: we're providing the id in the req.params, not the recipeName

File: backend/server.ts [hunk 2]
```diff
@@ -30,4 +30,20 @@ app.get("/gardens", async (req:Request , res:Response) => {
...
+  const description = req.body.description
```
Comment (NestorJMartinez) [inline backend/server.ts]: for this endpoint, we're assuming everything attribute in Garden is getting updated, so update all the attributes from #1

**Review by NestorJMartinez — CHANGES_REQUESTED** [review]
Make those changes, fix the merge conflict and I'll take another look. Thanks Storm!

File: backend/server.ts (hunk 2 above)
Comment (StormyW618) [reply backend/server.ts]: Hi I made the changes requested and re committed the updated files!

**Review by NestorJMartinez — APPROVED** [review]
looks good!

**Pull request merged**